### PR TITLE
Set default LLM to the last one used

### PR DESCRIPTION
**Pull request by pkukielka**
## Changes

User LLM selection is now saved and used for subsequest chats/edits.
TODO: new API for commands is needed to suport the same.

## Test plan

Manual tests according to QA guide

### CODE REVIEW

File: src/main/kotlin/com/sourcegraph/cody/edit/FixupService.kt [hunk 1]
```diff
@@ -53,10 +50,6 @@ class FixupService(val project: Project) : Disposable {
     return true
   }
 
-  fun setCurrentModel(model: ChatModelsResponse.ChatModelProvider) {
```
Comment (pkukielka) [inline src/main/kotlin/com/sourcegraph/cody/edit/FixupService.kt:56]: Value never used.

File: src/main/kotlin/com/sourcegraph/cody/edit/EditCommandPrompt.kt [hunk 2]
```diff
@@ -48,7 +48,7 @@ class EditCommandPrompt(val controller: FixupService, val editor: Editor, val di
       LlmDropdown(
           modelUsage = ModelUsage.EDIT,
           project = controller.project,
-          onSetSelectedItem = controller::setCurrentModel,
+          onSetSelectedItem = {},
```
Comment (mkondratek) [inline src/main/kotlin/com/sourcegraph/cody/edit/EditCommandPrompt.kt:51]: this callback should persist the default for the inline edits, right?

**Review by mkondratek — APPROVED** [review]
lgtm

File: src/main/kotlin/com/sourcegraph/cody/edit/EditCommandPrompt.kt (hunk 2 above)
Comment (pkukielka) [reply src/main/kotlin/com/sourcegraph/cody/edit/EditCommandPrompt.kt:51]: No, @danielmarquespt said the default between edits and chats should be the same.
We are already saving the default in `LlmDropdown::setSelectedItem`

**Pull request merged**